### PR TITLE
add test case

### PR DESCRIPTION
**Pull request by mdanny1209**

### CODE REVIEW

**Comment by call3hudson** [comment]
Could you explain why you did copy a same test case?

**Comment by mdanny1209** [comment]
> Could you explain why you did copy a same test case?

It's not the same one.
The added one is for double withdraw.

File: test/MasterChef.spec.ts [hunk 1]
```diff
@@ -199,6 +199,11 @@ describe('Masterchef', function () {
 
       expect(await sushi.balanceOf(user0.address)).to.equal(v10000.add(v10.mul(2)));
     });
+
```
Comment (call3hudson) [inline test/MasterChef.spec.ts:202]: if so, add relevant comment here, so forked users can easily get noticed

**Review by call3hudson — CHANGES_REQUESTED** [review]

File: test/MasterChef.spec.ts (hunk 1 above)
Comment (mdanny1209) [reply test/MasterChef.spec.ts:202]: Okay

**Pull request merged**